### ISSUE TITLE
Milestone 2 feedback

### ISSUE THREAD

**Issue by andytai7** [UBC-MDS/pyeasyeda#48]
Write test cases and code iteratively: Accuracy.

Comments
Great job! Each test was importing functions wrong; as a result, I had to drag and drop the coded part into the same directory. After this, I imported the code, and the tests worked fine. 

Write test cases and code iteratively: Code Quality.

Comments
All the functions and tests are documented (docstrings and comments are present)

We can install and use your Python package: Mechanics.

Comments
It would help if you linked your Github
The instructions:"pip install pyeasyeda" do not work
ERROR: Could not find a version that satisfies the requirement pyeasyeda (from versions: none)
ERROR: No matching distribution found for pyeasyeda

Instead, you should be using pip install git+(your git hub repo link)

Exception handling: Code Quality

Comments
The exception handling is included in the function’s body for all the functions.

**Comment by Luming-ubc** [comment]
Hi Andy,

Thanks for pointing out the misleading installations in the Readme file. However, I think we all tested our package by running the following two commands from the root of the our project repo. The test results are attached. We will update the usage for sure!! (Maybe it is because it was not mentioned in the milestone 2 checklist. :sweat_smile:)

Moreover, I don't quite understand what's wrong with the importing functions and why you would need to drag and drop the coded part, and which is the same directory? Please help clarify on these. Thanks for your efforts!

```
poetry install
poetry run pytest --cov=pyeasyeda

```
![image](https://user-images.githubusercontent.com/94151497/151288824-1241a041-fc0d-4b53-be02-6550ac67b028.png)

**Comment by andytai7** [comment]
Hi @Luming-ubc, 

Because of the confusing instructions in the read me, the tests that i ran, i had to modify the code. 

I believe the problem with having me move the files around to be able to run the tests is that the code within the tests do not call upon the functions from their directories. But while using your code, i could also run the function and tests! 

As a results, i gave part marks back. 

I hope this will suffice! 

Best, 
Andy

**Comment by andytai7** [comment]
<img width="530" alt="Screen Shot 2022-01-27 at 10 16 39 AM" src="https://user-images.githubusercontent.com/45893111/151419494-fbc92145-88b5-4c19-a12c-6d3c53b19cdb.png">

**Comment by khbunyan** [comment]
Hi @andytai7 

Thanks for the feedback. Still a bit lost on why it's not working on your end as expected (there was a comment by you with much lower coverage but I believe its now deleted?)

To test further I did the following:

- cloned a new repo
- created a new conda environment
-  used `pip install poetry`
-  then `poetry install`
-  then the `pytest tests/`
- then `pytest tests/ --cov=pyeasyeda`

The tests appear to be working and with much higher coverage, so I'm a bit confused how your tests are showing different values. Perhaps we could have a quick meeting to discuss? I've posted the results:

<img width="1032" alt="Screen Shot 2022-01-27 at 11 25 16 AM" src="https://user-images.githubusercontent.com/69618055/151429410-130e5419-9311-44f2-8e50-51dd42334afd.png">

Thank you
